opts.go

```diff
@@ -54,7 +53,7 @@ func WithProcessRunner(cmd *exec.Cmd) Opt {
 type WithSnapshotOpt func(*SnapshotConfig)
```

Comment:
not necessarily related to these changes, but is this needed? I don't see it being used anywhere

Reply:
Not in testing, but if you wanted to change an option (e.g. whether or not you want to resume the VM on snapshot load) that gives you the option to do so.

Reply:
Maybe it would be nice to have a test use this. Perhaps we can have the SSH test use the resume option.

Reply:
+1, @sondavidb can you create an issue that describes adding this test so we can track it?